obspy/core/inventory/inventory.py

```diff
@@ -666,6 +666,15 @@ def plot(self, projection='global', resolution='l',
 
         return fig
 
+    def plot_rays(self, *args, **kwargs):
+        """
+        Plot ray paths between this inventory and one or more events.
+
+        All options are passed to :func:`obspy.imaging.ray_paths.plot_rays`.
+        """
+        from obspy.imaging.ray_paths import plot_rays
+        plot_rays(inventory=self, *args, **kwargs)
```

Comment:
I would remove this - it is a bit redundant and just using the `plot_rays()` is almost as easy in my opinion. I know that this is how we did things in the past but we should probably try to keep our core classes as small as possible.